package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dist": "webpack --optimize-minimize",
     "compile": "babel src --out-dir lib",
-    "build": "npm run dist && npm run compile"
+    "build": "npm run dist && npm run compile",
+		"prepublish": "npm run compile"
```

Comment:
not a huge deal, but npm is militant about no tabs in package.jsons. I usually cave to its demands